Simple Incremental/Assets/Scripts/Scriptable/EnemyDamage.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+
+[CreateAssetMenu]
+public class EnemyDamage : ScriptableObject
+{
+    [SerializeField]
+    public  int minDamage;
+
+    [SerializeField]
+    public int maxDamage;
+
+    [SerializeField]
+    public float attackRange;
+
+    [SerializeField]
+    public float attackCooldown;
+
+    float damagePerSecond;

```

Comment:
I don't think the math on this is correct.

If my maximum damage is 50, and my minimum damage is 45, and my attackCooldown is 1 second, it would return 5dps.

Reply:
Additionally, I don't think you should have this variable defined. Instead, just return a float from your CalculateDamagePerSecond() method and use that directly instead. ScriptableObjects are shared across anyone who uses them.

Reply:
I understand the use case for ScirptableObjects. I'm unclear on the endstate here so I grouped that into the SO for now. That can definitely change depending on whether enemies will have inventory items or just some generic damage values.